RunningOrder/Sprint/SprintsView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  SprintsView.swift
+//  RunningOrder
+//
+//  Created by Clément Nonn on 07/07/2020.
+//  Copyright © 2020 Worldline. All rights reserved.
+//
+
+import SwiftUI
+
+extension Sprint: Identifiable {
+    var id: String { self.name }
+}
+
+struct SprintsView: View {
+    @State private var sprints: [Sprint] = []
+    @State private var showNewSprintModal = false
+
+    @Binding var selectedSprint: Sprint?
+
+    var body: some View {
+        List(selection: $selectedSprint) {
+            Section(header: Text("Active Sprints")) {
+                ForEach(sprints) { sprint in
+                    Text(sprint.name)
+                        .tag(sprint)
+                }
+
+                Button(action: { self.showNewSprintModal.toggle() }) {
```

Comment:
Do we really want to make the new sprint button part of the active sprint section? Could it deserve its own section?

Reply:
For UX consideration, I thought about it below or above all sections, but it didn’t fit well.. above is not good and what if there is lots of sprints ? Do we have to scroll all of it ? A floating button could be okay at the bottom, or inside the toolbar but for me it is okay to have here, Where we want to really add a sprint :)

Reply:
I'd go for a sticky button at the bottom :)